lib/fileio/jobs/readJob.cpp

```diff
@@ -60,23 +58,15 @@
 
 PXR_NAMESPACE_OPEN_SCOPE
 
-
-// for now, we hard code this to use displayColor.  But maybe the more
-// appropriate thing to do is just to leave shadingMode a lone and pass
-// "displayColor" in from the UsdMayaRepresentationFull
-// (usdMaya/referenceAssembly.cpp)
-const static TfToken ASSEMBLY_SHADING_MODE = UsdMayaShadingModeTokens->displayColor;
-
-
 UsdMaya_ReadJob::UsdMaya_ReadJob(
         const std::string &iFileName,
         const std::string &iPrimPath,
         const std::map<std::string, std::string>& iVariants,
         const UsdMayaJobImportArgs &iArgs) :
     mArgs(iArgs),
     mFileName(iFileName),
-    mPrimPath(iPrimPath),
     mVariants(iVariants),
+    mPrimPath(iPrimPath),
     mDagModifierUndo(),
     mDagModifierSeeded(false),
     mMayaRootDagPath()
```

Comment:
Given the ordering or properties in the header, these initializers need to be re-ordered to avoid tripping over `-Werror=reorder`:
```
    mArgs(iArgs),
    mFileName(iFileName),
    mVariants(iVariants),
    mMayaRootDagPath(),
    mPrimPath(iPrimPath),
    mDagModifierUndo(),
    mDagModifierSeeded(false)
```